installation/quickstart.rst

```diff
@@ -35,13 +40,6 @@ Rockstor `download options <https://rockstor.com/download.html>`_. Create a
 bootable CD or USB drive from the downloaded iso and proceed to the
 :ref:`installation` section.
 
-.. raw:: html
-
-   <div class="alert alert-info">
-   For convenience, you can also purchase a USB install disk from <a href="https://shop.rockstor.com/collections/diy-accessories/products/rockstor-usb-install-disk" target="_blank">our shop</a>.
-   Thanks for your support!
-   </div>
-
```

Comment:
Well that was a surprise. I was sure we had removed all of these links. Nice find / fix.